build-docker.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
+set -xe
+
+#
+## This script builds 3 types of Docker images (assuming version `VERSION`):
+#
+#   1. Image containing all qemu binaries. Built in `./containers/latest/`. Tagged with `:latest`, and `:VERSION`.
+#   2. Register image.  Built in `./containers/register/`. Tagged with `:register`, and `:register-VERSION`.
+#   3. A series of images, each bundled with a single guest-architecture qemu binary. Built in `./containers/qemu-ARCH`.
+#      Tagged with `:ARCH`, and `:ARCH-VERSION`.
+
+
+# Convert Travis-provided repo SLUG to lowercase - Docker's requirement for tags
+SLUG="$(echo "${TRAVIS_REPO_SLUG}" | tr '[:upper:]' '[:lower:]')"
```

Comment:
Please do not use `SLUG` (Github's owner_name/repo_name) as docker repo. The reason is as I wrote at other part.
Please keep using `DOCKER_REPO` in the file instead.